service/service_test.go

```diff
@@ -29,6 +29,8 @@ var (
 	tempAudioFUID       string
 	tempImageFUID       string
 	tempVideoFUID       string
+	validDesc           []byte
+	invalidDesc         []byte
```

Comment:
These buffers will hold the buffer for reading files with extremely long strings.